### PR TITLE
Implemented endpoint to add/remove torrents to/from a playlist

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis can you review? I will fix the line too long error in Pylint together with your feedback.

**Comment by devos50** [comment]
@lfdversluis ping

**Comment by lfdversluis** [comment]
@devos50 pong

File: Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py [hunk 1]
```diff
@@ -227,3 +230,119 @@ def render_POST(self, request):
...
+        """
+        channel_info = self.get_channel_from_db(self.cid)
+        if channel_info is None:
+            return ChannelsPlaylistsEndpoint.return_404(request)
+
+        playlist = self.channel_db_handler.getPlaylist(self.playlist_id, ['Playlists.dispersy_id'])
+        if playlist is None:
+            return BaseChannelsEndpoint.return_404(request, message="this playlist cannot be found")
+
+        channel_community = self.get_community_for_channel_id(channel_info[0])
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py]: Does this call do a DB call? If no, then maybe you could first do this check to save a query to the channel_db?

**Comment by lfdversluis** [comment]
Comments given

File: Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py (hunk 1 above)
Comment (devos50) [inline Tribler/Core/Modules/restapi/channels/channels_playlists_endpoint.py]: Good point, I was able to move this statement to another location, possibly saving a database query.

**Comment by devos50** [comment]
Job got aborted

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm ready for your review

**Pull request merged**